### DISCUSSION TITLE
Rotate Head Of Entity

### DISCUSSION THREAD

**Discussion by Codestech1** [Minestom/Minestom#1602] [Help] [answered]
Hi, i create mount on my server, how Can I rotate head of mount to be same as player?

**Comment by DevBlokJochem** [comment]
When you set the position of the entity, you can give a yaw and pitch too. You get:
```
new Pos(<x>, <y>, <z>, <yaw>, <pitch>);
```

**Reply by Codestech1** [reply]
Can i do it without teleporting? I wanna do it with animation.

**Comment by Codestech1** [comment, marked as answer]
`mount.setView(pos.yaw(), pos.pitch());`
It does what i want!
